### ISSUE TITLE
Character Creation Mode Property

### ISSUE THREAD

**Issue by Moonpile** [sun-dragon-cult/fvtt-system-rqg#632]
### _Why:_ Is your feature request related to a problem?

As a player or GM, I want there to be a Character Creation Mode that allows players to edit more of their character sheet.  This issue is just to create the property on the Actor to enable further development in later issues.

The property should be a enum to allow expansion to more view modes:
- Edit
- Play
- _CompactPlay_ (for a future alternative sheet that only displays a minimum of data)

## Main issue
- [ ]  #560 

## Blocks
- [ ] #635

**Comment by wake42** [comment]
I had a think about what this property should do - before I had an idea that it's actually a "view mode" so that it could contain `edit` | `play` | `compact play`. But maybe we don't need compact play if we make sure we make an integration module to the [Token HUD](https://foundryvtt.com/packages/token-action-hud-core) module. With that we should be able to access actions a token can do without even opening the actor sheet.

So in short I think a boolean is fine and the properly could maybe be called `editMode`.  When that is true we show input fields instead of text for chances in rune & skill tab as well as the characteristics values.
